File: domain/src/main/java/com/icehousecorp/maunorafiq/domain/city/repository/CityRepository.java
package com.icehousecorp.maunorafiq.domain.city.repository;

import com.icehousecorp.maunorafiq.domain.city.City;

import java.util.List;

import rx.Observable;

/**
 * Created by maunorafiq on 12/2/16.
 */

public interface CityRepository {

    boolean putCity(String city);

    Observable<List<City>> getCity();
}

File: domain/src/main/java/com/icehousecorp/maunorafiq/domain/city/repository/CityRepository.kt
package com.icehousecorp.maunorafiq.domain.city.repository

import com.icehousecorp.maunorafiq.domain.city.City

import rx.Observable

/**
 * Created by maunorafiq on 12/2/16.
 */

interface CityRepository {

    fun putCity(city: String): Boolean

    fun getCity(): Observable<List<City>>
}
